### ISSUE TITLE
Span has epoch as the startTime with PHP built-in webserver and cli

### ISSUE THREAD

**Issue by tmatsuo** [census-instrumentation/opencensus-php#131]
Here is the repro:

```
composer require google/cloud-trace opencensus/opencensus
```

```php
<?php

require_once __DIR__ . '/vendor/autoload.php';

use OpenCensus\Trace\Exporter\EchoExporter;
use OpenCensus\Trace\Tracer;

$exporter = new EchoExporter();
Tracer::start($exporter);

Tracer::inSpan(
    ['name' => 'slow_function'],
    function () {
        usleep(1);
    }
);
```

Run it with cli and built-in webserver. The inner span has 1970-01-01 00:00:00.000000 as the `startTime`.

**Comment by jcchavezs** [comment]
I think it is about cli config? Can you post the output of `php --ini` and maybe share your value for `Loaded Configuration File`?

**Comment by tmatsuo** [comment]
It's default php.ini installed by phpbrew. Which values do you think cause this?

**Comment by jcchavezs** [comment]
Check if `php-intl` is installed.
-- 
*José Carlos Chávez*
Spain: (+34) 672921339
Perú: (+51) 708-5422 Anexo 918

https://github.com/jcchavezs

**Comment by tmatsuo** [comment]
I installed ext-intl, but the result was the same

**Comment by chingor13** [comment]
I can reproduce this if the extension is enabled. @tmatsuo is that the case for you too?

**Comment by tmatsuo** [comment]
Yes

**Comment by chingor13** [comment]
The issue here was that the `ExtensionExporter` was supplying a `startTime` as `null` as an option to `opencensus_trace_begin`. The extension was taking the double value of null and setting it as the span's start time rather than grabbing the current timestamp.

This was fixed in 2 places (so can be fixed by updating either the extension or the php library).
* In the library, we set the default time to `microtime(true)` - available in composer version >= 0.2.3
* In the extension, we only take the provided `startTime` option if it's an int or double (assumed to be a unix timestamp) - available in PECL version >= 0.1.4
